source/frontend/src/features/properties/map/hooks/useMapProperties.tsx

```diff
@@ -15,11 +15,11 @@ export const useMapProperties = () => {
     (params?: IGeoSearchParams) => {
       const geoserver_params = {
         STREET_ADDRESS_1: params?.STREET_ADDRESS_1,
-        PID: params?.PID,
+        PID_PADDED: params?.PID,
```

Comment:
according to psp-6152 searching for 01 should match 236-626-010 as well as 010-463-251. The only way for that to be possible is to search against the padded pid instead of the pid (which is a number).